determined-ai/determined
# feat: Add tensorboard delete command to CLI

## Description

Creating a CLI command for deleting tensorboard files from a given experiment ID. We want some of the existing logic for checkpointGC to perform the file deleteion.

Usage of command would be `det tensorboards delete <experiment-id>`

## Test Plan

1. Create an experiment
Example: `det e create examples/tutorials/mnist_pytorch/const.yaml examples/tutorials/mnist_pytorch/`

2. Start Tensorboard and kill existing experiment:
`det tensorboard start <exp-id>`
`det e kill <exp-id>`

3. Verify that Tensorboard files exist:
```
$ cd /tmp/determined-cp/
$ find . -name 'tensorboard*' -type d
$ >> <folder-name>/tensorboard
$ cd <folder-name>/tensorboard/experiment/<exp-id>/trial/<exp-id>
$ ls
```

4. Delete tensorboard files:
`det tensorboard delete <exp-id>`

5. Check that files are deleted:
```
$ cd <folder-name>/tensorboard/experiment/
$ ls (folder for <exp-id> should be gone)
```

## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket

DET-9865
